perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -83,6 +84,20 @@
                      'Allow swiftclient to access Swift service without \n'
                      'having to verify the SSL certificate')
 
+flags.DEFINE_integer('objects_per_stream', 1000,
+                     'Number of objects to send and/or receive per stream. '
+                     'Only applies to the api_multistream scenario.',
+                     lower_bound=1)
+flags.DEFINE_integer('object_size_kb', 1,
```

Comment:
We discussed in person and the conclusion is we will use object_sizes as the flag name, for now it support one single size, and later it will be expanded to support a distribution.